Make image size height constraint when using the ImageSize.Large option

toger5 opened the pull request:
Fixes: https://github.com/vector-im/element-web/issues/19788
 - It adds a height check for "`height > width` images"
 - And, this skips the width constraint when using the Large setting. (always making the height now 480px (or the info image height) the dictating parameter)

turt2live reviewed (approved):
I suppose, though I thought this was fixed by Matthew's fix over the weekend? https://github.com/matrix-org/matrix-react-sdk/pull/7172

toger5 commented:
We changed different things. Matthew changed the defaults to be even larger. I fixed a bug where the height was not taken into consideration like seen in the linked issue: https://github.com/vector-im/element-web/issues/19788
